mixer/adapter/stackdriver/contextgraph/contextgraph.go

```diff
@@ -74,9 +73,10 @@ func (b *builder) Build(ctx context.Context, env adapter.Env) (adapter.Handler,
 	env.Logger().Debugf("Proj, zone, cluster, opts: %s,%s,%s,%s",
 		b.projectID, b.zone, b.cluster, opts)
 
-	meshUID := fmt.Sprintf("%s/%s/%s", b.projectID, b.zone, b.cluster)
-	if len(b.meshID) > 0 {
-		meshUID = b.meshID
+	meshUID := b.cfg.MeshUid
+	if len(meshUID) == 0 {
+		// must be a fallback here
+		meshUID = fmt.Sprintf("%s/%s/%s", b.projectID, b.zone, b.cluster)
```

Comment:
Why do we must have a fallback here? I think it makes more sense to set it only when it is explicitly set in handler.

Reply:
backwards compat. if we don't specify it in the handler, the intent is for the CG stuff to continue to work as it did before.

Reply:
Ok, let's hold on adding this to log, and only fill it in metric and edge.